extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/closed-caption/captionOpacityTab.js

```diff
@@ -0,0 +1,98 @@
+var React = require('react'),
+    ClassNames = require('classnames'),
+    Utils = require('../utils'),
+    CONSTANTS = require('../../constants/constants'),
+    SelectionContainer = require('./selectionContainer'),
+    Slider = require('../slider');
+
+var CaptionOpacityTab = React.createClass({
+
+  changeTextOpacity: function(event) {
+    if (!this.props.closedCaptionOptions.enabled) {
+      this.props.controller.toggleClosedCaptionEnabled();
+    }
+    var value = event.target.value;
+    this.props.controller.onClosedCaptionChange('textOpacity', value);
+  },
+
+  changeBackgroundOpacity: function(event) {
+    if (!this.props.closedCaptionOptions.enabled) {
+      this.props.controller.toggleClosedCaptionEnabled();
+    }
+    var value = event.target.value;
+    this.props.controller.onClosedCaptionChange('backgroundOpacity', value);
+  },
+
+  changeWindowOpacity: function(event) {
+    if (!this.props.closedCaptionOptions.enabled) {
+      this.props.controller.toggleClosedCaptionEnabled();
+    }
+    var value = event.target.value;
+    this.props.controller.onClosedCaptionChange('windowOpacity', value);
+  },
+
+  percentString: function(number) {
+    //if (number == 0) return "Transparent";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACommentedCode)